www/src/components/navigation.js

```diff
@@ -159,6 +159,7 @@ export default ({ pathname }) => {
           <NavItem linkTo="/features/">Features</NavItem>
           <NavItem linkTo="/blog/">Blog</NavItem>
           <NavItem linkTo="/showcase/">Showcase</NavItem>
+          <NavItem linkTo="/newsletter/">Newsletter</NavItem>
```

Comment:
A question for @fk - do we want to add this to the main nav? As we're fairly space constrained there.

Reply:
Unfortunately, we don't have space here -- especially with Site Showcase and Starter Showcase on the way

Reply:
👍 

@wuweiweiwu can you delete this line?

Reply:
yep!

Reply:
@m-allanson Re: adding items to the main navigation: https://github.com/gatsbyjs/gatsby/issues/6377#issuecomment-404114364 — maybe you have another idea?